pages/index.module.scss

```diff
@@ -16,3 +16,25 @@
     width: 100vw;
   }
 }
+
+.updates_container {
```

Comment:
noice 

[nit/personal take]: it's helpful to move all the styling into css modules when introducing it to a section that was using utliity classes previously